ui/packages/consul-ui/app/components/auth-profile/index.hbs

```diff
@@ -1,9 +1,12 @@
-<dl>
+<dl
+  class="auth-profile"
+  ...attributes
+>
     <dt>
       <span>My ACL Token</span><br />
       AccessorID
     </dt>
     <dd>
-      {{substr item.AccessorID -8}}
+      {{string-substring @item.AccessorID (sub @item.AccessorID.length 8)}}
```

Comment:
`substr` is one of our own app helpers that we own. I'd rather we used the ember-string-fns addon provided alternative here (so we can eventually delete our own `substr` one once we use it no more)